index.js

```diff
@@ -31,10 +31,10 @@ module.exports = ({ dynamicAssetPrefix = false, ...nextConfig } = {}) => {
             options: {
               limit: nextConfig.inlineImageLimit,
               fallback: require.resolve("file-loader"),
-              publicPath: `${nextConfig.assetPrefix || nextConfig.basePath}/_next/static/images/`,
+              publicPath: `${(dynamicAssetPrefix ? '' : nextConfig.assetPrefix) || nextConfig.basePath}/_next/static/images/`,
```

Comment:
I think we can remove this `nextConfig.basePath` from this line. Because I think `basePath` is for links, not for assets which we're trying to solve with this package. What do you think?

Reply:
If no static asset prefix is provided and dynamic asset prefix is disabled we should still fallback to `basePath` becuase assets will be served under the `basePath` if it is specified. But if `assetPrefix` is set or `dynamicAssetPrefix` is truish then we don't have to take `basePath` into consideration: in this case static/dynamic `assetPrefix` should contain the `basePath` part too. I fix the condition here to represent this.